src/Aquarius.ONE.ClientSDK/Utilities/UI/Attribute.cs

```diff
@@ -1,16 +0,0 @@
-﻿using System;
-using System.Collections.Generic;
-using System.Text;
-
-namespace ONE.Utilities.UI
-{
-    public class Attribute

```

Comment:
Now that this has been removed the `Attribute` reference in https://github.com/AquaticInformatics/aquarius-one-sdk/blob/9440b70c819401a8cc2bf1249a7b57bd941d5cb1/src/Aquarius.ONE.ClientSDK/Utilities/UI/UIDefinition.cs#L23 has been silently replaced with `System.Attribute` (or it always referenced that).